ancpbids/plugins/plugin_dsvalidator.py

```diff
@@ -17,9 +18,12 @@ def execute(self, dataset, report: ValidationPlugin.ValidationReport):
                 val = getattr(obj, name)
                 use = member['use']
                 if (lb > 0 or use == 'required') and not val:
-                    report.error(f"Missing required field {name} at {top_path}.")
+                    # TODO: Identify more precisely where these errors emerge from
```

Comment:
if the obj variable is an instance of File or Folder, then top_path represents its path relative to the root directory, else from obj you can climb up its ancestor hierarchy until you found a File or Folder